Lib/fontmake/font_project.py

```diff
@@ -202,6 +202,8 @@ def save_otfs(
                 useProductionNames=use_production_names)
             otf.save(otf_path)
 
+            if subset is None:
+                subset = (GLYPHS_PREFIX + 'Keep Glyphs') in ufo.lib
```

Comment:
I guess we'd also want to subset if for any glyph's lib the key `GLYPHS_PREFIX + 'Glyphs.Export'` maps to `False`.